content/docs/reference/pkg/_index.md

```diff
@@ -95,11 +95,13 @@ Each cloud vendor has a dedicated package for deploying resources to it:
 * [**DNSimple** (`@pulumi/dnsimple`)]({{< relref "nodejs/pulumi/dnsimple" >}})
 * [**Fastly** (`@pulumi/fastly`)]({{< relref "nodejs/pulumi/fastly" >}})
 * [**F5 BigIP** (`@pulumi/f5bigip`)]({{< relref "nodejs/pulumi/f5bigip" >}})
+* [**GitHub** (`@pulumi/github`)]({{< relref "nodejs/pulumi/github" >}})
```

Comment:
Is the intention that this PR would be merged _after_ the docs for GitHub and MongoDB Atlas are merged? Because listing them here will lead users to a 404 without the actual docs merged.

Reply:
GitHub is already merged - there’s a PR open for mongodbatlas now